.circleci/config.yml

```diff
@@ -1,236 +1,66 @@
-          steps:
-            - run:
-                name: Coverage
-                command: |
-                  poetry run coverage xml --data-file .coverage
-            - codecov/upload:
-                file: coverage.xml
-
-  integration-test-parallel:
-    parameters:
-      python-version:
-        type: string
-        default: "3.9"
-      pytest-group:
-        type: string
-        default: "generic"
-    docker:
-      - image: cimg/python:<< parameters.python-version >>
+  setup:
+    executor: path-filtering/default
     resource_class: small
-    environment:
-      POETRY_CACHE_DIR: /home/circleci/project/.poetry
-    parallelism: 4
     steps:
       - checkout
-      - restore_cache:
-          key: ci-cache-<< parameters.python-version >>-{{ checksum "pyproject.toml" }}
       - run:
-          name: Run << parameters.pytest-group >> integration tests
-          command: |
-            export KOLENA_TOKEN=${KOLENA_TOKEN}
-            export KOLENA_CLIENT_BASE_URL=${KOLENA_CLIENT_BASE_URL}
-            TEST_GROUP="<< parameters.pytest-group >>"
-            TESTFILES=$(circleci tests glob tests/integration/$TEST_GROUP/*.py |
-              circleci tests split --split-by=timings --timings-type=filename)
-            poetry run pytest -vv --durations=0 --cov=kolena --cov-branch --junitxml=test-results/result.xml \
-                $TESTFILES
-      - when:
-          # Generate coverage only from one python version
-          condition:
-            equal: [ "3.9", << parameters.python-version >> ]
-          steps:
-            - run:
-                name: Coverage
-                command: |
-                  poetry run coverage xml --data-file .coverage
-            - codecov/upload:
-                file: coverage.xml
-            - store_test_results:
-                path: test-results
-
-  integration-test:
-    parameters:
-      python-version:
-        type: string
-        default: "3.9"
-      pytest-group:
-        type: string
-        default: "generic"
-    docker:
-      - image: cimg/python:<< parameters.python-version >>
-    resource_class: small
-    environment:
-      POETRY_CACHE_DIR: /home/circleci/project/.poetry
-    steps:
-      - checkout
-      - restore_cache:
-          key: ci-cache-<< parameters.python-version >>-{{ checksum "pyproject.toml" }}
+          name: Set base-revision
+          command: echo "export BASE_REVISION=$(git merge-base trunk << pipeline.git.revision >>)" >> $BASH_ENV
+      - path-filtering/set-parameters:
+          base-revision: ${BASE_REVISION}
+          mapping: |
+            kolena/classification/.* classification true
+            tests/integration/classification/.* classification true
+            kolena/detection/.* detection true
+            tests/integration/detection/.* detection true
+            kolena/fr/.* fr true
+            tests/integration/fr/.* fr true
+            kolena/workflow/.* generic true
```

Comment:
Non-blocking, but I've been meaning to change the `generic` integration tests to `workflow` to reflect this external naming convention.

Reply:
https://github.com/kolenaIO/kolena/pull/46